acs5zippoverty_child.py

```diff
@@ -0,0 +1,67 @@
+import json
+from census_response import getCensusResponse
+
+def main():
```

Comment:
Same comments from the county script.

I made a single script for zip and county data for the race data, we might want to do that here?

Reply:
If you have one script that can do both, we should use that for all of them. 
Is that what you use in the file `acs5racedemographics.py`?

Reply:
Yeah. One problem with the script is that it should really call two different functions for the different geographies instead of passing a string to one function, but we can change that later if needed.

We could probably put everything in one script by passing a dictionary with the name and tables we want, but for this PR let's just merge the geographies.

Reply:
Rather than make the above changes to the county_child and zip_child files, I will submit a new PR using the method in the race data script, yes?

Reply:
You can push to your branch and it will update the PR

Reply:
Which change I am making before I push to my branch?

Reply:
I'd push all the changes to this PR.